source/server-discovery-and-monitoring/tests/monitoring/replica_set_with_removal.yml

```diff
@@ -52,6 +37,25 @@ phases:
           server_opening_event:
             topologyId: "42"
             address: "b:27017"
+  -
```

Comment:
It looks like this change moves `responses` from the first object in the `phases` array to the second. Previously, the first object has both `responses` and `outcome` fields and the second object had only an `outcome` field.

This doesn't jive with the test README, which says that each object in the `phases` array has two keys. Could we add some language explaining that `responses` is optional?

Reply:
Added.